### PR TITLE
feat: Add more labels to match for classification

### PR DESCRIPTION
**Pull request by KillianCourvoisier**

### CODE REVIEW

File: src/metadata.js [hunk 1]
```diff
@@ -43,23 +43,37 @@ function appendMetadata(docs) {
 }
 
 function evalClassification(label) {
+  // const wordsArray = label.split(' ')
```
Comment (LucsT) [inline src/metadata.js:46]: Need to remove this  2 comments

File: src/metadata.js [hunk 2]
```diff
@@ -43,23 +43,37 @@ function appendMetadata(docs) {
...
+  // const firstThreeWords = `${wordsArray[0]} ${wordsArray[1]} ${wordsArray[2]}`
   if (
     // Here for unknown reason, sometimes we find "Avis de dégrèvement" and sometimes "Avis dégrèvement"
     // We have to cover both possibilities
     label.match(
-      /Avis (de situation|de taxes?|d'impôt|de dégrèvement|dégrèvement|supplémentaire)/
+      /Avis (de situation|de taxes?|d'impôt|de dégrèvement|dégrèvement|supplémentaire|impôts sur)/
```
Comment (LucsT) [inline src/metadata.js:52]: Seems a duplicate with L55 to me, no ?
'Avis impôts sur'

File: src/metadata.js [hunk 3]
```diff
@@ -43,23 +43,37 @@ function appendMetadata(docs) {
...
     return 'other_tax_document'
-  } else if (label.match(/^Avis échéancier/)) {
+  } else if (
+    label.match(/^Avis échéancier|Échéancier/) ||
+    label.match(/Prélèvement mensuel/)
+  ) {
     return 'tax_timetable'
   } else {
     log('warn', 'Impossible to evalute Classification metadata for one doc')
+    // Keeping this log for further potentials untreated labels
```
Comment (LucsT) [inline src/metadata.js:75]: To remove if you remove the two first comments

**Review by LucsT — APPROVED** [review]

File: src/metadata.js (hunk 2 above)
Comment (KillianCourvoisier) [reply src/metadata.js:52]: right, eagle eye :+1:

**Pull request merged**